### PR TITLE
9 drag the file to the screen

### LINKED ISSUE #9: drag the file to the screen

### PR DESCRIPTION
**Pull request by jonyyy1**

### CODE REVIEW

File: web/static/scripts.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+async function uploadFile(event) {
+            event.preventDefault();
+            const fileInput = document.getElementById('file');
+            const file = fileInput.files[0];
+            const chunkSize = 1024 * 1024; // 1MB
+            const totalChunks = Math.ceil(file.size / chunkSize);
+            let fileId = null;
+            const csrfToken = document.getElementById('csrf_token').value;
+
```
Comment (ThePeruvianFox) [inline web/static/scripts.js]: creas el master

File: web/static/scripts.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                formData.append('chunk_number', i);
+                formData.append('total_chunks', totalChunks);
+                formData.append('file_name', file.name);
+
+                if (fileId) {
+                    formData.append('file_id', fileId);
+                }
+
+                try {
+                    const response = await fetch('http://localhost:8000/upload_file_chunk', {
```
Comment (ThePeruvianFox) [inline web/static/scripts.js]: upload/chunkedfile

Comment (ThePeruvianFox) [reply web/static/scripts.js]: {
masterid: pk
}

File: web/static/scripts.js (hunk 1 above)
Comment (ThePeruvianFox) [inline web/static/scripts.js]: axios.create(master) -> obj masterfile

id = masterfile.id

Comment (jonyyy1) [reply web/static/scripts.js]: listo!!

File: web/static/scripts.js (hunk 2 above)
Comment (jonyyy1) [reply web/static/scripts.js]: listoo!!!

File: web/static/scripts.js (hunk 1 above)
Comment (jonyyy1) [reply web/static/scripts.js]: listoo!!1

**Review by ThePeruvianFox — APPROVED** [review]
lgtm

**Pull request merged**